toolsrc/src/vcpkg/help.cpp

```diff
@@ -174,8 +174,9 @@ namespace vcpkg::Help
             // clang-format off
             "  vcpkg edit <pkg>                Open up a port for editing (uses " ENVVAR(EDITOR) ", default 'code')\n"
             "  vcpkg import <pkg>              Import a pre-built library\n"
-            "  vcpkg create <pkg> <url> [archivename]\n"
-            "                                  Create a new package\n"
+            "  vcpkg create <pkg> <url> <archivename/REF>\n"
+            "               [TRPLET(windows/linux/osx)\n"
```

Comment:
```suggestion
            "               [TRIPLET(windows/linux/osx)\n"
```

Also shouldn't this use our normal `--triplet` ?

Reply:
@BillyONeal Since the build method under each triplet may be different, we need to pass in multiple triplet parameters here.